manifests/server/database.pp

```diff
@@ -2,14 +2,13 @@
 define postgresql::server::database(
   $comment          = undef,
   $dbname           = $title,
-  $owner            = $postgresql::server::user,
+  $owner            = undef,
   $tablespace       = undef,
   $template         = 'template0',
   $encoding         = $postgresql::server::encoding,
   $locale           = $postgresql::server::locale,
   $istemplate       = false,
   $connect_settings = $postgresql::server::default_connect_settings,
-  $change_ownership = false,
```

Comment:
Removing this parameter is a breaking change that might cause existing configurations to break. It is better for users to accept the parameter, but not act on it, except for emitting a warning that it will go away. Newer versions of stdlib have a `deprecation` method to do so in a friendly way.